### DISCUSSION TITLE
English (or french, or italian) user interface

### DISCUSSION THREAD

**Discussion by Virgus** [jeessy2/ddns-go#798] [General]
Hello and thanks for this project !
I just discovered it in CasaOS app store and installed it, but I'm having an hard time since i cannot read Chinese.
Is there an option or a command line to change the GUI language somehow ?

Thanks and have a nice day,
V.

**Comment by jeessy2** [comment]
Unfortunately, DDNS-GO only supports chinese at the moment. It may be possible to support english in the future.

**Reply by Virgus** [reply]
Thanks, looking forward to test future "international" version as soon as it will be available.

**Comment by jeessy2** [comment]
Hello. ddns-go v6.0.0 is already supports english. you can try it.
